.gitignore

```diff
@@ -24,3 +24,6 @@ venv/
 !.pre-commit-config.yaml
 !.readthedocs.yml
 !.renovaterc.json
+
+# Exclude license file/s
+python-gitlab-ci.gitlab-license
```

Comment:
Interesting, this hasn't been an issue for us, do you have leftover licenses in the project workspace when running this?

Reply:
This could be due to my ignorance in reading/finding out how to properly test enterprise features. I saw somewhere in docker compose to add `python-gitlab-ci.gitlab-license` file at the root and it worked hence added an `.gitignore` entry to avoid accidentally pushing it.

Reply:
Oh, this should be automatically handled by the test setup. It will generate a temporary license for testing. But if that doesn't work just let us know.

Reply:
oh ok! I didn't know that tests are generating enterprise license automatically. That is really cool :D